servlet.py

```diff
@@ -1304,6 +1311,40 @@ def sanity_check():
         sys.exit(1)
 
 
+def apply_config(args, apySection):
+    for (name, value) in vars(args).items():
+        if name in apySection:
+            # Get default from private variables of argparse
+            default = None
+            for action in parser._actions:
+                if action.dest == name:
+                    default = action.default
+
+            # Try typecasting string to type of argparse argument
+            fn = type(value)
+            res = None
+            try:
+                if fn is type(None):
+                    if apySection[name] == 'None':
+                        res = None
+                    else:
+                        res = apySection[name]
+                elif fn is bool:
+                    if apySection[name] == "False":
+                        res = False
+                    elif apySection[name] == "True":
+                        res = True
+                    else:
+                        res = bool(apySection[name])
+                else:
+                    res = fn(apySection[name])
+            except ValueError:
+                pass
```

Comment:
Should we print some sort of warning here?

Reply:
How about `print("Unable to cast " + apySection[name] + " to expected type")` ?